Description: Make sure desktop systems know how to pass files to lmms
Author: Petter Reinholdtsen <pere@debian.org>
Forwarded: no, should be forwarded
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2014-10-07

--- lmms-1.0.3.orig/data/lmms.desktop
+++ lmms-1.0.3/data/lmms.desktop
@@ -6,7 +6,7 @@ GenericName[de]=Software zur Musik-Produ
 Comment=easy music production for everyone!
 Comment[ca]=Producció fàcil de música per a tothom!
 Icon=lmms
-Exec=env QT_X11_NO_NATIVE_MENUBAR=1 lmms
+Exec=env QT_X11_NO_NATIVE_MENUBAR=1 lmms %f
 Terminal=false
 Type=Application
 Categories=Qt;AudioVideo;Audio;Midi;
